basis/locals/parser/parser.factor

```diff
@@ -31,15 +31,16 @@ ERROR: invalid-local-name name ;
 
 SINGLETON: lambda-parser
 
-: ((parse-lambda)) ( assoc reader-quot: ( -- quot ) -- quot )
+: with-lambda-scope ( assoc reader-quot: ( -- quot ) -- quot )
     '[
         in-lambda? on
         lambda-parser quotation-parser set
-        [ use-words @ ] [ unuse-words ] bi
+        manifest [ clone [ clone ] change-qualified-vocabs ] change
```

Comment:
Seems like a clean fix -- another option would be to define ``M\ manifest clone`` and all four containers in it (``search-vocab-names``, ``search-vocabs``, ``qualified-vocabs``, ``auto-used``).  On the other hand, this is a simple specific fix which I like.